### ISSUE TITLE
Error running timer `org-pomodoro-tick': (void-variable org-pomodoro-sound-finished-args)

### ISSUE THREAD

**Issue by dsdshcym** [lolownia/org-pomodoro#38]
Since version 2.0.2 I think, org-pomodoro-timer stops at "00:01" and shows this message every time.

```
Error running timer `org-pomodoro-tick': (void-variable org-pomodoro-sound-finished-args)
```

This makes org-pomodoro nearly unusable cause the timer will not stop automatically.

Maybe the typo isn't fixed completely?

**Comment by EricTing** [comment]
+1

**Comment by leoc** [comment]
I merged #40 which is doing the same fix. Still, thank you very much! :+1:
